### DISCUSSION TITLE
TypeError while using useIsMutation

### DISCUSSION THREAD

**Discussion by vineetdigit** [TanStack/query#2535] [Q&A] [answered]
Hello, 

What could be the reason for getting this error which points to line where I am calling the useIsMutation hook?

TypeError: Object(...) is not a function

```
import { useIsMutating } from "react-query";

function SomeComponent() {
    const isMutating = useIsMutating();

...
}
```
thanks,

Vineet

**Comment by TkDodo** [comment, marked as answer]
not sure, I just tried it in a codesandbox and it worked fine: https://codesandbox.io/s/use-is-mutating-gihhr

**Reply by vineetdigit** [reply]
Weird. And for some reason, useIsMutating doesn't appear in the list of hooks. I just upgraded to react-query 3.19.1, but still no luck.

<img width="416" alt="Screen Shot 2021-08-04 at 12 53 01 AM" src="https://user-images.githubusercontent.com/11036997/128143476-fb9d9cd0-0cfb-497e-8d9d-a4134e348b61.png">

**Reply by TkDodo** [reply]
hmm it's definitely here:

https://github.com/tannerlinsley/react-query/blob/07b9f2ce97bc94907c66564190f83166994995d5/src/react/index.ts#L11

have you tired nuking your node-modules ?

**Reply by vineetdigit** [reply]
yes, I checked `/node_modules/react-query/dist/react-query.development.js` and the symbol is exported.

You were right. I had to nuke the package and reinstall it. Now it has worked. I'm so sorry for the noise. :-( 

Thanks @TkDodo

**Reply by dmarshall83** [reply]
I'm getting this same issue. I have tried to uninstall the library and reinstall and I still get the same error. @vineetdigit is there something more I need to do than just 'npm uninstall react-query' before I reinstall?

**Reply by vineetdigit** [reply]
@dmarshall83 Not really. 

I initially simply upgraded the package (`npm install react-query`), which didn't help. Then I uninstalled the package and reinstalled it. And it helped. After this, I deliberately restarted the app with the npm script (`react-scripts start`)
